File: ncDados/dCliente.vb
Namespace nsCliente

    Public Class ColecaoCliente
        Inherits List(Of dCliente)
    End Class

    Public Class dCliente

        Private _cid As Nullable(Of Integer)
        Private _codigo As String
        Private _nome As String
        Private _endereco As String
        Private _estadoCivil As String
        Private _sexo As String
        Private _nomePai As String
        Private _nomeMae As String
        Private _situacao As String
        Private _rg As String
        Private _rgOrgaoEmissor As String
        Private _rgUf_cid As Nullable(Of Integer)
        Private _cpf As String
        Private _carteiraProfissional As String
        Private _dataNascimento As String
        Private _naturalidade As String
        Private _nacionalidade As String
        Private _dataInclusao As String
        Private _email As String
        Private _foto As String
        Private _ddd As String
        Private _telefone As String
        Private _dddcel As String
        Private _celular As String
        Private _veiculo As String

        Public Property cid() As Nullable(Of Integer)
            Get
                Return _cid
            End Get
            Set(ByVal value As Nullable(Of Integer))
                _cid = value
            End Set
        End Property

        Public Property codigo() As String
            Get
                Return _cid
            End Get
            Set(ByVal value As String)
                _cid = value
            End Set
        End Property

        Public Property nome() As String
            Get
                Return _nome
            End Get
            Set(ByVal value As String)
                _nome = value
            End Set
        End Property

        Public Property endereco() As String
            Get
                Return _endereco
            End Get
            Set(ByVal value As String)
                _endereco = value
            End Set
        End Property

        Public Property dataNascimento() As String
            Get
                Return _dataNascimento
            End Get
            Set(ByVal value As String)
                _dataNascimento = value
            End Set
        End Property

        Public Property naturalidade() As String
            Get
                Return _naturalidade
            End Get
            Set(ByVal value As String)
                _naturalidade = value
            End Set
        End Property

        Public Property nacionalidade() As String
            Get
                Return _nacionalidade
            End Get
            Set(ByVal value As String)
                _nacionalidade = value
            End Set
        End Property

        Public Property estadoCivil() As String
            Get
                Return _estadoCivil
            End Get
            Set(ByVal value As String)
                _estadoCivil = value
            End Set
        End Property

        Public Property sexo() As String
            Get
                Return _sexo
            End Get
            Set(ByVal value As String)
                _sexo = value
            End Set
        End Property

        Public Property nomePai() As String
            Get
                Return _nomePai
            End Get
            Set(ByVal value As String)
                _nomePai = value
            End Set
        End Property
        Public Property nomeMae() As String
            Get
                Return _nomeMae
            End Get
            Set(ByVal value As String)
                _nomeMae = value
            End Set
        End Property
        Public Property situacao() As String
            Get
                Return _situacao
            End Get
            Set(ByVal value As String)
                _situacao = value
            End Set
        End Property

        Public Property dataInclusao() As String
            Get
                Return _dataInclusao
            End Get
            Set(ByVal value As String)
                _dataInclusao = value
            End Set
        End Property
        Public Property rg() As String
            Get
                Return _rg
            End Get
            Set(ByVal value As String)
                _rg = value
            End Set
        End Property
        Public Property rgOrgaoEmissor() As String
            Get
                Return _rgOrgaoEmissor
            End Get
            Set(ByVal value As String)
                _rgOrgaoEmissor = value
            End Set
        End Property
        Public Property rgUf_cid() As Nullable(Of Integer)
            Get
                Return _rgUf_cid
            End Get
            Set(ByVal value As Nullable(Of Integer))
                _rgUf_cid = value
            End Set
        End Property
        Public Property cpf() As String
            Get
                Return _cpf
            End Get
            Set(ByVal value As String)
                _cpf = value
            End Set
        End Property

        Public Property carteiraProfissional() As String
            Get
                Return _carteiraProfissional
            End Get
            Set(ByVal value As String)
                _carteiraProfissional = value
            End Set
        End Property
        Public Property email() As String
            Get
                Return _email
            End Get
            Set(ByVal value As String)
                _email = value
            End Set
        End Property

        Public Property foto() As String
            Get
                Return _foto
            End Get
            Set(ByVal value As String)
                _foto = value
            End Set
        End Property

        Public Property ddd() As String
            Get
                Return _ddd
            End Get
            Set(ByVal value As String)
                _ddd = value
            End Set
        End Property

        Public Property telefone() As String
            Get
                Return _telefone
            End Get
            Set(ByVal value As String)
                _telefone = value
            End Set
        End Property

        Public Property dddcel() As String
            Get
                Return _dddcel
            End Get
            Set(ByVal value As String)
                _dddcel = value
            End Set
        End Property

        Public Property celular() As String
            Get
                Return _celular
            End Get
            Set(ByVal value As String)
                _celular = value
            End Set
        End Property

        Public Property veiculo() As String
            Get
                Return _veiculo
            End Get
            Set(ByVal value As String)
                _veiculo = value
            End Set
        End Property

    End Class

End Namespace

File: ncModelos/dCliente.cs
using System;
using System.Collections.Generic;

namespace nsCliente
{
    public class ColecaoCliente : List<dCliente>
    {
    }

    public class dCliente
    {
        private int? _cid;
        private string _codigo;
        private string _nome;
        private string _endereco;
        private string _estadoCivil;
        private string _sexo;
        private string _nomePai;
        private string _nomeMae;
        private string _situacao;
        private string _rg;
        private string _rgOrgaoEmissor;
        private int? _rgUf_cid;
        private string _cpf;
        private string _carteiraProfissional;
        private string _dataNascimento;
        private string _naturalidade;
        private string _nacionalidade;
        private string _dataInclusao;
        private string _email;
        private string _foto;
        private string _ddd;
        private string _telefone;
        private string _dddcel;
        private string _celular;
        private string _veiculo;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string nome
        {
            get => _nome;
            set => _nome = value;
        }

        public string endereco
        {
            get => _endereco;
            set => _endereco = value;
        }

        public string dataNascimento
        {
            get => _dataNascimento;
            set => _dataNascimento = value;
        }

        public string naturalidade
        {
            get => _naturalidade;
            set => _naturalidade = value;
        }

        public string nacionalidade
        {
            get => _nacionalidade;
            set => _nacionalidade = value;
        }

        public string estadoCivil
        {
            get => _estadoCivil;
            set => _estadoCivil = value;
        }

        public string sexo
        {
            get => _sexo;
            set => _sexo = value;
        }

        public string nomePai
        {
            get => _nomePai;
            set => _nomePai = value;
        }

        public string nomeMae
        {
            get => _nomeMae;
            set => _nomeMae = value;
        }

        public string situacao
        {
            get => _situacao;
            set => _situacao = value;
        }

        public string dataInclusao
        {
            get => _dataInclusao;
            set => _dataInclusao = value;
        }

        public string rg
        {
            get => _rg;
            set => _rg = value;
        }

        public string rgOrgaoEmissor
        {
            get => _rgOrgaoEmissor;
            set => _rgOrgaoEmissor = value;
        }

        public int? rgUf_cid
        {
            get => _rgUf_cid;
            set => _rgUf_cid = value;
        }

        public string cpf
        {
            get => _cpf;
            set => _cpf = value;
        }

        public string carteiraProfissional
        {
            get => _carteiraProfissional;
            set => _carteiraProfissional = value;
        }

        public string email
        {
            get => _email;
            set => _email = value;
        }

        public string foto
        {
            get => _foto;
            set => _foto = value;
        }

        public string ddd
        {
            get => _ddd;
            set => _ddd = value;
        }

        public string telefone
        {
            get => _telefone;
            set => _telefone = value;
        }

        public string dddcel
        {
            get => _dddcel;
            set => _dddcel = value;
        }

        public string celular
        {
            get => _celular;
            set => _celular = value;
        }

        public string veiculo
        {
            get => _veiculo;
            set => _veiculo = value;
        }
    }
}
